### ISSUE TITLE
message.sender.id isn't returning anything valid

### ISSUE THREAD

**Issue by StephanieSunshine** [obrie/turntabler#7]
Hi, based on the example in modlist.rb message.sender.id isn't working for me.

```
if moderator_ids.include?(message.sender.id) && message.content =~ /^\/mod$/
```

Which as a test in my code I did:

```
   client.on :user_spoke do |message|
     if message.content =~ /bop/
     client.room.current_song.vote   # => true
   client.room.say("Smoke!");
   end
   if message.content =~ /^\/hugs/
    client.room.say("Awww Thank you #{client.user.name}")
    client.room.say("Hugs #{client.user.name}")
   end
   if message.content =~ /^\/hello$/
     puts "/hello received"
     client.room.say("#{client.room.current_song.title} Awesomes: #{client.room.current_song.up_votes_count} Lames: #{client.room.current_song.down_votes_count}")
   end

   if message.content =~ /^\/hopup$/
     puts "/hopup received"
     puts "#{message.sender.id}"
   end
```

_snip_

to which i get the error

D, [2013-01-08T10:02:29.086367 #2088] DEBUG -- : Message received: {"command"=>"speak", "userid"=>"xxxxxxxxxxxxxxxxxxxxxxxxxxx", "name"=>"xxxxxxxxxxxxxxxx", "roomid"=>"xxxxxxxxxxxxxxxxxxxxxxxxxx", "text"=>"/hopup"}
/hopup received
E, [2013-01-08T10:02:29.087215 #2088] ERROR -- : undefined method `id' for nil:NilClass
./seriousBot.rb:65:in`block (2 levels) in <main>'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/handler.rb:51:in `call'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/handler.rb:51:in`block in run'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/handler.rb:49:in `each'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/handler.rb:49:in`run'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/client.rb:453:in `block in on_message'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/client.rb:452:in`each'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/client.rb:452:in `on_message'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/client.rb:102:in`block in connect'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/connection.rb:130:in `call'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler/connection.rb:130:in`block in dispatch'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler.rb:89:in `call'
/var/lib/gems/1.9.1/gems/turntabler-0.1.3/lib/turntabler.rb:89:in`block (2 levels) in run'
/var/lib/gems/1.9.1/gems/em-synchrony-1.0.2/lib/em-synchrony.rb:26:in `call'
/var/lib/gems/1.9.1/gems/em-synchrony-1.0.2/lib/em-synchrony.rb:26:in`block (2 levels) in synchrony'

Hopefully this is an easy fix?

**Comment by jasonhazel** [comment]
This is related to the Message object not correctly handling user information when dealing with chat messages.  This has been fixed as of this morning I believe and a new gem should be out soon.

**Comment by obrie** [comment]
This should be fixed in 0.1.4 -- looks like you're still on 0.1.3 based on the output.  @mrhazel thanks for helping out!

**Comment by StephanieSunshine** [comment]
Cool thank you that did fix it
